CONTRIBUTORS.md

```diff
@@ -0,0 +1,9 @@
+# Contributors
+
+People who have contributed to the js-data-datastore project. This file is controlled by scripts/contributors.js
+
+Commit count | Name | Email
+---:|---|---
+8 | Dariusz Dusza | dariusz@vpscash.nl
+5 | Marco | mvallotto.dev@google.com
+18 | Robin Bron | robin@finwo.nl
```

Comment:
I don't see the scripts folder anymore. Are you sure this still belongs in the repository?